firefox-ios/firefox-ios-tests/Tests/XCUITests/IntegrationTests.swift

```diff
@@ -155,9 +155,9 @@ class IntegrationTests: BaseTestCase {
 
         // Sync again just to make sure to sync after new name is shown
         app.buttons["Settings"].waitAndTap()
-        mozWaitForElementToExist(app.staticTexts["ACCOUNT"])
-        app.tables.cells.element(boundBy: 2).waitAndTap()
-        mozWaitForElementToExist(app.tables.staticTexts["Sync Now"], timeout: TIMEOUT_LONG)
+        navigator.nowAt(SettingsScreen)
+        navigator.goto(BrowserTab)
+        waitForInitialSyncComplete()
```

Comment:
@skhamis Even after the 2nd `waitForInitialSyncComplete()`, I still encounter an error on TPS:
```
2025-03-31 00:39:13.802 CROSSWEAVE INFO: executing action VERIFY on tab {"uri":"https://example.com/","profile":"Fennec (cso) on iOS"}
2025-03-31 00:39:13.806 CROSSWEAVE ERROR: [phase phase1] RunNextTestAction failed - Error: ASSERTION FAILED! error locating tab(resource://tps/logger.sys.mjs:76:11) JS Stack trace: AssertTrue@logger.sys.mjs:76:11
```

Reply:
After signing in before opening example.com, I no longer encounter this error from TPS. 😄